Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -36,7 +36,10 @@ public class ResourcesProjectMapper: ProjectMapping { // swiftlint:disable:this
         var additionalTargets: [Target] = []
         var sideEffects: [SideEffectDescriptor] = []
 
-        let bundleName = "\(project.name)_\(target.name.replacingOccurrences(of: "-", with: "_"))"
+        let sanitizedTargetName = target.name
+            .replacingOccurrences(of: "-", with: "_")
+            .replacingOccurrences(of: "/", with: "_")
```

Comment:
Given in how many places we needed to update this, I wonder if we should extract this into a helper method such as:
```swift
extension TuistGraph.Target {
  static func sanitizedTargetName(_ targetName: String) -> String { ... }
}
```